qiskit/_result.py

```diff
@@ -414,3 +414,22 @@ def get_qubitpol_vs_xval(self, nqubits, xvals_dict=None):
                     circuit_name, z_dicts[qubit_ind])
 
         return qubitpol, xvals
+
+
+def copy_qasm_from_qobj_into_result(qobj, result):
```

Comment:
shouldn't this be a private method?

Reply:
I don't think so. It is related to the `Result` type but it does not seem like an operation on the `Result` type. Ideally, it should be placed near the simulators which are the ones using it but I did not want to add a new file nor I was sure about where to add it.